### DISCUSSION TITLE
`PointerReleased` event triggered to fast, when using the touch screen

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#14522] [Q&A]
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/3e9bc603-66d8-4293-97e2-3e894940bde6)

I create a headcontrol, and add event
```
private void HeadControl_PointerPressed(object? sender, PointerPressedEventArgs e)
{
    (VisualRoot as Window)?.BeginMoveDrag(e);
}
```
And then the window cannot be dragged and the event is triggered prematurely

**Comment by maxkatz6** [comment]
I don't think our approach BeginMoveDrag even works with WM_POINTER OS events.

**Reply by Coloryr** [reply]
ok.....
